stock_quant_sale_order/i18n/ja.po

```diff
@@ -1,116 +1,125 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-# * stock_quant_sale_order
+# 	* stock_quant_sale_order
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 11.0\n"
+"Project-Id-Version: Odoo Server 15.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-01-17 03:15+0000\n"
-"PO-Revision-Date: 2018-01-17 03:15+0000\n"
-"Last-Translator: <>\n"
+"POT-Creation-Date: 2023-10-30 03:28+0000\n"
+"PO-Revision-Date: 2023-10-30 03:28+0000\n"
+"Last-Translator: \n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
 #. module: stock_quant_sale_order
-#: model:ir.ui.view,arch_db:stock_quant_sale_order.quant_sale_order_wizard_form
+#: model_terms:ir.ui.view,arch_db:stock_quant_sale_order.quant_sale_order_wizard_form
 msgid "Cancel"
-msgstr "取消"
+msgstr ""
```

Comment:
@AungKoKoLin1997 Please help revive the translation where the original string hasn't changed.